pypesto/objective/amici_calculator.py

```diff
@@ -89,14 +92,31 @@ def __call__(self,
             parameter_mapping=parameter_mapping,
             amici_model=amici_model
         )
+        rdatas = []
```

Comment:
Just above, the paramteer mapping needs to be chunked as well... So this must be moved inside the whole chunking logic